bundle/manifests/runtime-component.clusterserviceversion.yaml

```diff
@@ -487,7 +487,13 @@ spec:
           - deployments
           - statefulsets
           verbs:
-          - '*'
+          - create
```

Comment:
There are 2 more wildcards in RuntimeOperation [controller](https://github.com/application-stacks/runtime-component-operator/blob/main/controllers/runtimeoperation_controller.go).